openreview/venue_request/venue_request.py

```diff
@@ -710,6 +711,46 @@ def setup_venue_deployment(self):
                 }
             ))
 
+    def setup_venue_post_submission(self):
+
+        post_submission_content = {
+            'force': {
+                'value-radio': ['Yes', 'No'],
+                'required': True,
+                'description': 'Force to create the blind submission if conference if double-blind'
```

Comment:
What do you think about: `Force creating blind submissions if conference is double blind`

Reply:
Fixed.